### PR TITLE
Writing away `import $rdf` to prepare for next version of rdflib

### PR DESCRIPTION
**Pull request by megoth**
As rdflib now moves to import/export we should refactor our code so that it makes use of the new possibilities, e.g. treeshaking.

This PR fixes this for all TS-based files.

Should not be merged before https://github.com/linkeddata/rdflib.js/pull/335 is merged into rdflib and pushed to NPM

### CODE REVIEW

File: typings/solid-namespace/index.d.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 declare module 'solid-namespace' {
   import { NamedNode } from 'rdflib'
 
+  type RDFLibSubset = { namedNode: (value: string) => NamedNode }
   type toNamedNode = (label: string) => NamedNode
   export type Namespaces = {[alias: string]: toNamedNode}
 
-  export default function vocab(rdflib: typeof import('rdflib')): Namespaces
+  export default function vocab(rdf: RDFLibSubset): Namespaces
```
Comment (RubenVerborgh) [inline typings/solid-namespace/index.d.ts:8]: You likely want to rely on the `DataFactory` export of `rdflib`, which has standard RDF/JS typings.

File: trustedApplications/trustedApplicationsUtils.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import $rdf, { NamedNode, IndexedFormula, Statement } from 'rdflib'
+import { NamedNode, IndexedFormula, Statement, BlankNode, st, sym } from "rdflib"
```
Comment (RubenVerborgh) [inline trustedApplications/trustedApplicationsUtils.ts]: Might want to rely on `DataFactory` instead

File: typings/solid-namespace/index.d.ts (hunk 1 above)
Comment (megoth) [reply typings/solid-namespace/index.d.ts:8]: I will do this after fixing the typings for rdflib, which exposes DataFactory as a namespace (which cannot be used as specific type in TypeScript).

Comment (megoth) [reply typings/solid-namespace/index.d.ts:8]: Actually, based on this, maybe we should try to expose typings file in rdflib? So we don't have to go through https://github.com/DefinitelyTyped/DefinitelyTyped to make these changes. [I added an issue on rdflib for this](https://github.com/linkeddata/rdflib.js/issues/341).

File: scratchpad/__snapshots__/data.test.ts.snap [hunk 3]
```diff
@@ -23,7 +23,7 @@ Array [
   Statement {
     "object": Literal {
       "termType": "Literal",
-      "value": "Scratchpad (1970-1-1)",
+      "value": "Scratchpad (1/1/1970)",
```
Comment (TallTed) [inline scratchpad/__snapshots__/data.test.ts.snap]: I don't know what the mistake here was, but...

Ambiguous date patterns are problematic everywhere.  `1/1/1970` is unambiguous by chance; `12/10/1970` is not.  `1970-1-1` is troublesome, but `1970-01-01` at least *suggests* that this app's date-based file-naming pattern might be ISO- and xsd-compliant, i.e., YYYY-MM-DD (which I hope it is).

**Review by Vinnl — APPROVED** [review]

**Review by timbl — APPROVED** [review]
Looks good and rdflib now is published as needed at 1.0.3 
Go ahead

**Pull request merged**